<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="settingsFrameLayout">
      <item>
       <widget class="QLabel" name="settingsLabel">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="settingsFormLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="layoutLabel">
          <property name="text">
           <string>Layout</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="outputTypeLabel">
          <property name="text">
           <string>Output Type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="outputTypeComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>*.png</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>*.odt</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>*.docx</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="formatLabel">
          <property name="text">
           <string>Format</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="formatComboBox">
          <item>
           <property name="text">
            <string>Code 128</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EAN 128</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="layoutFrame">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="barcodeCheckbox">
             <property name="text">
              <string>Print Barcode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="nameCheckbox">
             <property name="text">
              <string>Print Name</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
